modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the Kubernetes control plane
+
+To configure a TLS security profile for the Kubernetes control plane, edit the `APIServer` custom resource (CR).  If a TLS profile is not configured, the default TLS profile is `Intermediate`. The Kubernetes control plane consists of the Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager.
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the the minimum TLS version and the TLS ciphers required by the Kubernetes control plane components to communicate with each other.
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile  in the `APIServer` custom resource (CR) under `Spec.Tls Security Profile` and `Status.Tls Profile`. 
+
+[IMPORTANT]
+====
+The Kubernetes control plane does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3` it is not supported. The Kubernetes control plane converts the `Modern` profile to `Intermediate`.
```

Comment:
I think you can remove this whole note here. Per @stlaz the API server doesn't do the 1.0 -> 1.1 conversion for the old profile. And it doesn't do the 1.3->1.2 conversion either.

We've already noted that the modern profile isn't supported in the overview section, so I don't think we need to mention it again here.

Reply:
@bergerhoffer Should we leave _The Kubernetes control plane does not support TLS `1.3`_?

Reply:
@stlaz I believe you told me that the API server didn't convert 1.3 to 1.2 (for custom profiles). Can you confirm that 1.3 is actually supported for the control plane?